packages/gestalt/src/Badge.css

```diff
@@ -3,10 +3,17 @@
   composes: rounding1 from "./Borders.css";
   align-content: center;
   display: flex;
-  padding: var(--space-0) 6px;
```

Comment:
I think 6px should be a component token as `--badge-padding-left`

Reply:
we don't have cmp token for spacing yet.. i think